tests/unit/elements/Button.spec.js

```diff
@@ -0,0 +1,90 @@
+import { shallowMount } from '@vue/test-utils'
+
+import Button from '@/elements/Button/Button'
+
+describe('Button.vue', () => {
+    const ROUNDNESS = 'complete'
+    const COLOR = 'red'
+    const ICON = 'vote-yea'
+    const SIZE = 'mega'
+    const INDICATION = 'positive'
+    const IS_INVERTIBLE = true
+    const IS_CALL_TO_ACTION = true
+    const SLOT = '<strong class="test-my-slot">Hello World</strong>'
+    let defaultConfig = null
+
+    beforeEach(() => {
+        defaultConfig = {
+            propsData: {
+            }
+        }
+    })
+
+    function getWrapper() {
+        return shallowMount(Button, defaultConfig)
+    }
+
```

Comment:
Order tests based on the order of mixins and props in the component

Reply:
Solved.